### ISSUE TITLE
Note accidental might be 'undefined' sometimes

### ISSUE THREAD

**Issue by eagor** [saebekassebil/teoria#107]
```
var scale = teoria.scale("cx", 'wholetone');
scale.simple();
> ["cx", "dx", "ex", "fundefined", "gundefined", "aundefined"]
```
I suspect this is a result of accidental()
```
scale.notes()[3].accidentalValue()
> 3
scale.notes()[3].accidental()
> undefined
```
But the accidentalValue should not be greater than 2. If it is, then it should be another note name. Thus I think the root of the issue is in the way the intervals are calculated, particularly, how it finds the note that is some interval away from another note (from scale tonic in our case).

**Comment by eagor** [comment]
so this is it:
```
teoria.note("cx").interval("A4").toString()
>"fundefined3"
```
but seems to be like note name has to be F (as 4th note after C)
what does music theory has to answer to this:
C## + A4 = ?

**Comment by saebekassebil** [comment]
Yes. that's the problem. I don't think there's an answer for that - since we generally speaking never deal in "triple accidentals" like that.

I think throwing an error would be best - since you're trying to do something "impossible"
